index.js

```diff
@@ -1,12 +1,19 @@
 'use strict';
-module.exports = (arr, comp) => {
-	for (let i = 0; i < arr.length - 1; i++) {
-		const a = arr[i];
-		const b = arr[i + 1];
 
-		if (comp ? comp(a, b) > 0 : a > b) {
+const defaultComparator = (a, b) => a > b ? 1 : 0;
+module.exports = (arr, comp) => {
+	// pass only iterable objects
+	if (!arr || typeof arr[Symbol.iterator] !== 'function') {
+		return false;
+	}
```

Comment:
I agree, but this is an important condition, otherwise if the input comes non-iterable or empty object, we get a failure